LunchAgentService/Services/SchedulerService.cs

```diff
@@ -33,21 +36,24 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
                     continue;
                 }
 
-                Log.LogDebug("Getting menus");
+                try
+                {
+                    if (DateTime.Now.Hour == 10)
```

Comment:
Combination of this condition and a delay at line 59 will cause this to post menus 3/4 times per day. We need to refactor this entire method